utils/check_repo.py

```diff
@@ -206,6 +206,8 @@
     "FlaxCLIPTextModel",
     "FlaxCLIPVisionModel",
     "FlaxWav2Vec2ForCTC",
+    "FlaxXLMRobertaForCausalLM",
+    "TFXLMRobertaForCausalLM",
```

Comment:
Oh those should be put in the auto-mappings. You just have to set them in `models.auto.modeling_flax_auto` and `models.auto.modeling_tf_auto`.